### PR TITLE
CVSL-1563 Connect PP caseload to hard stop screens

### PR DESCRIPTION
**Pull request by SebNegahbanMoJ**

### CODE REVIEW

File: server/views/pages/create/caseload.njk [hunk 1]
```diff
@@ -11,11 +11,10 @@
 {% set noJsBackLink = true %}
 {% set showCommsBanner = showWhatsNewBanner %}
 
-{% macro offenderName(offenderName, crn, prisonerNumber, licenceId, isClickable, index) %}
+{% macro offenderName(offenderName, crn, prisonerNumber, licenceId, isClickable, index, createLink) %}
     <div class="caseload-offender-name">
-        {% set link = "/licence/create/id/" + licenceId + "/check-your-answers" if licenceId else "/licence/create/nomisId/" + prisonerNumber + "/confirm"%}
         {% if isClickable %}
-            <a id="name-button-{{ index }}" href="{{ link }}" class="govuk-link govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</a>
+            <a id="name-button-{{ index }}" href="{{ createLink }}" class="govuk-link govuk-heading-s govuk-!-padding-left-0 govuk-!-margin-bottom-0">{{ offenderName }}</a>
```
Comment (andrewrlee) [inline server/views/pages/create/caseload.njk:17]: love it 🤩

**Review by andrewrlee — DISMISSED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by gurps-singh — DISMISSED** [review]

**Review by nishanthmahasamudram — APPROVED** [review]
LGTM

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**